Update README.md (#353)
Altinn/altinn-access-management-frontend: README.md

   dotnet run
   ```

> - Add below text in your hosts file on your pc/mac. Path to file on mac: /private/etc/hosts. Path to file on Windows: c:\Windows\System32\Drivers\etc\hosts.

Reviewer: ```suggestion
- Add the following to your hosts file. Path to file on mac: /private/etc/hosts. Path to file on Windows: c:\Windows\System32\Drivers\etc\hosts.
```
